jekyll/_ccie/preview.md

```diff
@@ -278,26 +265,26 @@ No changes, the same process as for the CCIE 1.0 <https://circleci.com/docs/ente
          ]
      }
      ```
-   * EC2 security group should have these rules:
+7. Configure the EC2 security group to have the following rules:
 
      | Protocol | Port Range | Source  |
      | -------- | ---------- | ------- |
      | TCP      | 22         | 0.0.0.0 |
      | TCP      | 2376       | 0.0.0.0 |
      | TCP      | 32768-61000| 0.0.0.0 |
-
+     {: class="table table-striped"}
 
 ## Upgrading MongoDB
 
-CircleCI Enterprise 2.0 runs MongoDB 3.2.11 which uses WiredTiger as the storage engine. If you are upgrading an existing database (Your existing 1.0 install or any Preview version from `Preview3` or older) you must use the following steps to upgrade your Mongo database. 
+CircleCI Enterprise 2.0 runs MongoDB 3.2.11 which uses WiredTiger as the storage engine. If you are upgrading an existing database, for example, your existing 1.0 install or any Preview version from `Preview3` or older, you must use the following steps to upgrade your Mongo database. 
```

Comment:
This feels like it has way too many commas, but I don't have a suggestion for improvement.